torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1215,7 +1210,10 @@ class InsertQuantDeQuantHelper {
   // Map from Graph to observer node, we can use observer node to
   // get the information of original value that's been observed and
   // the quantization parameters
-  std::unordered_map<Graph*, std::vector<Node*>> observer_nodes_;
+  std::unordered_map<Graph*, std::vector<Node*>> observer_nodes_for_graph_;
+  // A map from qparam name (e.g. _scale) to the attribute name in
```

Comment:
Can you update the comment to reflect the true nature of the map? Seems to be a map of observer node to the map you describe above.